app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Work < ApplicationRecord
+  validates :title, presence: true
+
+  has_many :votes
+  has_many :users, through: :votes
+
+  def self.top_ten(media)
+    results = Work.where(media: media.to_s)
+    results = results.sort_by {|result| result.votes.length}.reverse
+
+    results = Work.where(media: media.to_s) if results.empty?
+
+    if results.length <= 10
+      return results
+    else
+      results = results[0..9]
```

Comment:
In Ruby, slicing the array with `[0..9]` actually works out in your favor. Consider that taking out the `if ... else` and just doing `return results[0..9]` makes your tests pass